server/hap2/hatohol/hap2_fluentd.py

```diff
@@ -50,6 +51,7 @@ def set_arguments(self, args):
         self.__launch_args = args.fluentd_launch.split(" ")
         self.__accept_tag_reg = args.tag
         self.__accept_tag_pattern = re.compile(self.__accept_tag_reg)
+        self.__log_status_interverl = args.log_status_interval
```

Comment:
It should be fix Hap2Fluentd in test. Because for now, it does not specify `arg.log_status_interval`: https://github.com/project-hatohol/hatohol/blob/b2ea925a6d8c78ba5d6dfcd723576f73c894fb63/server/hap2/hatohol/test/TestHap2Fluentd.py#L176-L185